### ISSUE TITLE
Missing preposition in a few location strings

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#326]
**Missing prepositions**
- [x] Preposition missing in release phrase when a release happens at the wall. Hearing: 
"No change in position. Yellow balloon, wall." Should be "No change in position. Yellow balloon, at wall."

- [x] Preposition missing in jump alert when I use JN hotkey with the wall removed. I hear "right side of Play Area", but should hear "On right side of Play Area."

**Comment by jessegreenberg** [comment]
This has been fixed @terracoda thanks for the report.
